packages/react-composites/src/composites/CallComposite/pages/DtmfDialpadPage.tsx

```diff
@@ -103,7 +101,5 @@ const DtmfDialerContentTimer = (): JSX.Element => {
  * @internal
  */
 export const DtmfDialpadPage = (props: DialpadPageProps): JSX.Element => {
-  const adapter = useAdapter();
-
-  return <DtmfDialpadPageContent adapter={adapter} {...props} mobileView={props.mobileView} />;
+  return <DtmfDialpadPageContent {...props} mobileView={props.mobileView} />;
```

Comment:
Guess this doesn't need the adapter passed through?

Reply:
Didn't understand why it was written the way it was, `DtmfDialpadPageContent` can just use useAdapter itself and not have it passed through, so moved it